components/SatisfactionSlider.tsx

```diff
@@ -39,11 +39,22 @@ const SatisfactionSlider = ({
 
   return (
     <>
-      {title && <h3>{title}</h3>}
       <fieldset className={styles.wrapper} disabled={disabled}>
+        {title && <legend id={`${title}-legend`}>{title}</legend>}
         <Smiley percentage={scaledNumber} />
-        <input name={title} type="hidden" value={scaledNumber} />
+        <input aria-hidden="true" name={title} type="hidden" value={number} />
         <input
+          aria-label={
+            title +
+            ', ' +
+            min +
+            ' is the worst rating, ' +
```

Comment:
These need to be localized! I'm not sure our internationalization framework supports variables either. Are you sure we need a label since we have all the other Aira attributes correct?

Reply:
Good catch about localization. We might not need it, will remove